CompoundV3-Bots/Reserves-Monitor/src/agent.spec.ts

```diff
@@ -0,0 +1,337 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleBlock,
+  BlockEvent,
+} from "forta-agent";
+import { RESERVES_ABI, TARGET_RESERVES_ABI } from "./constants";
+
+import {
+  createAddress,
+  TestBlockEvent,
+  MockEthersProvider,
+} from "forta-agent-tools/lib/tests";
+import { BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import Fetcher from "./dataFetcher";
+
+const TEST_ADDRESSES = [createAddress("0x11"), createAddress("0x12")];
```

Comment:
Try using addresses that include letters so you can catch problems with address comparison (like `"0x000000000000000000000000000000000000def1" !== "0x000000000000000000000000000000000000dEf1"`).